src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/ElasticSearchExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// -------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Elasticsearch.Net;
+using Microsoft.Diagnostics.EventFlow.Configuration;
+
+namespace Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch
+{
+    internal static class ElasticSearchExtensions
```

Comment:
Is there a particular reason why you used extension methods instead of just adding them to `ElasticSearchOutputConfiguration` class?

If the answer is "no", I suggest to keep things simple

Reply:
Sounds good to me. My thinking was that the configuration class was meant to be a POCO of some type. I will move these into the suggested class.